### DISCUSSION TITLE
Hellix not display errors or warnings correctly inside zellij

### DISCUSSION THREAD

**Discussion by jgardona** [helix-editor/helix#8778] [General]
### Summary

Hellix not display errors or warnings correctly inside zellij. This happens when running helix inside zellij and you got an issue that can be a warning or an error from the compiler. The markers are not appearing as we see in print.

![image](https://github.com/helix-editor/helix/assets/146892539/fdadd4a2-bd8a-4ad9-90da-48853863cc1b)

### Reproduction Steps

Open zellij then run helix inside.

### Helix log

No issues in log

### Platform

Linux

### Terminal Emulator

alacritty

### Installation Method

got an appimage

### Helix Version

 23.10

**Comment by pascalkuthe** [comment]
Zellij doesn't properly implement the undercurl/colored underline escape sequences and I guess the terminfo detection fails because TERM is still set by the host terminal (which does support it).


Not much we can do about that in the helix side. The solution is for zellij to implement support for those escape codes.

**Reply by the-mikedavis** [reply]
Also see https://github.com/zellij-org/zellij/issues/2179. This may have been fixed in Zellij very recently in https://github.com/zellij-org/zellij/pull/2918
